libs/ledger-live-common/src/families/bitcoin/datasets/bitcoin.ts

```diff
@@ -238,6 +239,107 @@ const dataset: CurrenciesData<Transaction> = {
             };
           },
         },
+        {
+          name: "Send with OP_RETURN",
+          transaction: (t): Transaction => {
+            return {
+              ...t,
+              family: "bitcoin",
+              recipient: "BC1QQMXQDRKXGX6SWRVJL9L2E6SZVVKG45ALL5U4FL",
+              amount: new BigNumber(500),
+              opReturnData: Buffer.from("charley loves heidi", "utf-8"),
+              feePerByte: new BigNumber(1),
+              networkInfo: {
+                family: "bitcoin",
+                feeItems: {
+                  items: [
+                    {
+                      key: "0",
+                      speed: "high",
+                      feePerByte: new BigNumber(3),
+                    },
+                    {
+                      key: "1",
+                      speed: "standard",
+                      feePerByte: new BigNumber(2),
+                    },
+                    {
+                      key: "2",
+                      speed: "low",
+                      feePerByte: new BigNumber(1),
+                    },
+                  ],
+                  defaultFeePerByte: new BigNumber(1),
+                },
+              },
+              rbf: false,
+              utxoStrategy: {
+                strategy: 0,
+                excludeUTXOs: [],
+              },
+            };
+          },
+          expectedStatus: () => {
+            return {
+              // FIXME: this test is ran twice and OpReturnData is undefined the 2nd time
```

Comment:
Interesting, do you know why?

Reply:
I tried to investigate but didn't found the root cause
I think several checks in this file are commented as well for this reason